kin-sdk-core/src/main/java/kin/sdk/core/KinClient.java

```diff
@@ -20,85 +23,142 @@
      */
     public KinClient(Context context, ServiceProvider provider) {
         this.clientWrapper = new ClientWrapper(context, provider);
+        keyStore = clientWrapper.getKeyStore();
     }
 
     @VisibleForTesting
     KinClient(ClientWrapper clientWrapper) {
         this.clientWrapper = clientWrapper;
+        keyStore = clientWrapper.getKeyStore();
     }
 
     /**
-     * Create the account if it hasn't yet been created.
-     * Multiple calls to this method will not create an additional account.
+     * Create the account or returns the already created at index 0.
+     * <p>Multiple calls to this method will not create an additional account, use {@link #addAccount(String)} for
+     * creating multiple accounts.</p>
      * Once created, the account information will be stored securely on the device and can
      * be accessed again via the {@link #getAccount()} method.
      *
      * @param passphrase a passphrase provided by the user that will be used to store the account private key securely.
-     * @return {@link KinAccount} the account created
-     * @throws CreateAccountException if go-ethereum was unable to generate the account (unable to generate new key or
-     * store the key).
+     * @return {@link KinAccount} the account created store the key).
      */
-    public KinAccount createAccount(String passphrase) throws CreateAccountException {
```

Comment:
let's check if we can remove the method instead of having deprecated.
And we need to keep `CreateAccountException`

Reply:
will be discussed separately, in the meantime we keep it as deprecated